README.md

```diff
@@ -23,13 +23,32 @@ nix copy \
   --to "http://localhost:9000?compression=none" $storePath
 ```
 
-#### Note on compression
-While it's also possible to upload with narfile compression, this is only used
-to compress *uploads* into the binary cache.
+### Binary Cache
+As of now, `nix-casync` can be used as a space-efficient binary cache.
 
-`nix-casync` will decompress compressed narfile uploads on the fly, chunk the
-uncompressed payload, and serve those as `/nar/$narhash.nar`. It will also
-rewrite uploaded `.narinfo` files accordingly.
+You probably want to put some reverse proxy doing SSL in front of it, and add
+some protection on the `PUT` endpoints.
+
+The following section describes some internal behaviour of `nix-casync`, and
+how it treats Narfiles and Narinfo files.
```

Comment:
```suggestion
how it treats NAR files and Narinfo files.
```
It seems to be a pattern, Is that on purpose?

Reply:
The codebase also uses `Narfile` consistently, mostly because I didn't want to write NAR File and NAR Info File.